app-api-common/recipe/src/main/java/org/codeforvictory/android/recipes/data/api/recipe/RecipeApiSource.kt

```diff
@@ -0,0 +1,17 @@
+package org.codeforvictory.android.recipes.data.api.recipe
+
+import org.codeforvictory.recipes.domain.common.model.meal.Meal
+
+class RecipeApiSource(
+    private val adaptToMeal: AdaptToMeal = AdaptToMeal(),
```

Comment:
It's weird to me to have to inject/instantiate what really amounts to a pure, static function (conversion).

Reply:
See my comment: https://github.com/CodeForVictory/recipes-mvi-playground/pull/13#discussion_r567547895